### DISCUSSION TITLE
update handle and edges

### DISCUSSION THREAD

**Discussion by zx405** [bcakmakoglu/vue-flow#1499] [Q&A]
One of my nodes has a handle on the left and I dynamically update the handle position to the top so why do my lines still come out of the left handle position when they start linking and I make sure the view is updated

**Comment by bcakmakoglu** [comment]
[Read this section of the docs, please.](https://vueflow.dev/guide/handle.html#dynamic-handle-positions-adding-removing-handles-dynamically)

**Reply by zx405** [reply]
Yes, I called this method but I looked at all my data and found that the values were all right whether it was location or id or location but as soon as I clicked on the handle to connect I found that the line was coming out of the old position After I call the connect-start method and print the parameters inside, I find that handleId is indeed the old id. I wonder if there is some hidden data controlling this wrong connection

**Reply by bcakmakoglu** [reply]
I'm sorry I wasn't able to follow what you described.

Can you provide a reproduction of the issue? [You can find a template here.](https://codesandbox.io/p/devbox/vue-flow-basic-gfgro4)
